grafana-plugin/src/style/vars.css

```diff
@@ -38,7 +38,7 @@
 }
 
 .theme-light {
-  --cards-background: var(--blue-2);
+  --cards-background: rgba(244, 245, 245);
```

Comment:
This was correct for theme-dark but wrong value for theme-light as it was pointing to blue instead. It had no use prior to this, changed the value of it